templates/chat.mustache

```diff
@@ -0,0 +1,54 @@
+<div id="webchat" role="main"></div>
+
+<script crossorigin="anonymous" src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
+
+<script>
```

Comment:
Does this work in a JavaScript block, too?
```<!-- HTML here -->>
{{^element.frozen}}
{{#js}}
// JavaScript code...
{{/js}}
{{/element.frozen}}
```

Reply:
Yes, it works in a JavaScript block

Reply:
It would be nice if we would use them)  as the Moodle Mustache docs states:

> If this template is rendered by PHP, the JavaScript is separated from the HTML, and is appended to a special section in the footer of the page after requirejs has loaded. This provides the optimal page loading speed.

However, we could postpone this change in a separate PR, if you prefer)